api/v1/version_service.proto

```diff
@@ -19,41 +19,47 @@ package api.v1;
 
 import "validate/validate.proto";
 
-option go_package = "github.com/koor-tech/version-service/api/v1";
+option go_package = "github.com/koor-tech/version-service/api/v1;apiv1";
```

Comment:
Is the `apiv1` still needed or shouldn't this be removed as the folder is called `v1` only anymore as a package. The `go.mod` file is removed in your PR